core/arch/arm/kernel/link_dummies_paged.c

```diff
@@ -32,9 +35,21 @@ __thread_std_smc_entry(uint32_t a0 __unused, uint32_t a1 __unused,
 {
 	return 0;
 }
-void __section(".text.dummy.__wq_rpc")
-__wq_rpc(uint32_t func __unused, int id __unused,
-	 const void *sync_obj __unused, const char *fname __unused,
-	 int lineno  __unused)
-{
-}
+
+const struct mobj_ops mobj_reg_shm_ops __rodata_dummy;
```

Comment:
minor: what about using alphabetical ordering for this definition series.

Reply:
Or perhaps sorting it based on line length? (Just kidding) :-)
That's a bit much in my opinion, the order of these has no impact.

Reply:
ok :)

Reply:
Why does it matter to put all these symbols in `.rodata.dummy`? In other words, what happens if `__rodata_dummy` is not added?

Reply:
Then they'd end up with the original section name and get pulled into the unpaged section for that reason. Now for these structs it's wouldn't be so bad since we're anyway forcing them into the unpaged section, but I still think it's important that it ends up in the unpaged section for the right reason.

Reply:
Oh, to be honest I had somewhat forgotten (again) about the link process. Got it, thanks.

Reply:
> Oh, to be honest I had somewhat forgotten (again) about the link process. Got it, thanks.

Understandable, it is complicated. The good news is that it usually work well enough to be temporarily forgotten. :-)